doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -200,6 +200,9 @@ Bluetooth samples
   * Added configuration for the nRF5340 target.
   * Fixed scanning start on the nRF5340 target with the Zephyr LL controller.
     Previously, it was not possible to start scanning, because the :kconfig:option:`CONFIG_BT_EXT_ADV` option was disabled for the Zephyr LL controller.
+  * :ref:`ble_nrf_dm` sample:
+
+    * Added support for the nRF5340.
```

Comment:
```suggestion
    Previously, it was not possible to start scanning, because the :kconfig:option:`CONFIG_BT_EXT_ADV` option was disabled for the Zephyr LL controller.
    
* :ref:`ble_nrf_dm` sample:

    * Added support for the nRF5340 target.
```